packages/react-pose/src/components/PoseElement/index.tsx

```diff
@@ -172,7 +172,8 @@ class PoseElement extends React.PureComponent<PoseElementInternalProps> {
     const refs: RefSetters = {};
     const { elementType } = this.props;
 
-    if (typeof elementType === 'string' || isForwardRef(elementType)) {
+    if (typeof elementType === 'string') {
+      //  || isForwardRef(elementType) TODO: Fix types for isForwardRef
```

Comment:
technically this is a breaking change that got released as patch

Reply:
IMO a breaking change is something that is documented API that changes, or you could consider it a bug as we technically supported it and then broke it rather than purposefully removed it. I don't even think it was released as it isn't in any patch notes.

I'll take a look at adding it back but there's no official typings and I was having trouble getting Rollup to respect my custom ones.